components/ClassInviteTable.js

```diff
@@ -351,7 +351,7 @@ export default function ClassInviteTable({
               {classes.description}
             </h1>
           </div>
-          <Link href={`/dashboard/${classes.classroomId}`} passHref>
+          <Link href={`/dashboard/v2/${classes.classroomId}`} passHref>
             <button className='border-2 border-fcc-gray-15 bg-fcc-gray-90 text-white font-bold py-2 px-4 rounded'>
```

Comment:
Hi @utsab and @GuillermoFloresV accidentally added this as part of the function description pull request - but this file change should address not having to manually add 'v2' to the url now.